### DISCUSSION TITLE
DS918+ with  WAVLINK adapter working but only at 1000 Mbps and 45 MB/s

### DISCUSSION THREAD

**Discussion by johnrues** [bb-qq/r8152#450] [Q&A] [answered]
Hi,

For some reason, I only get 1000 Mbps in the Network Interface and 45 MB/s transfer rate
My configuration is:

- DS918+ with DSM version 7.2.2-72806.
- WAVLINK 5Gbps USB C RTL8157, https://www.amazon.com/dp/B0DBZ41BPL
- The WAVLINK adapter is plugged in the back.
- The USB C to USB A adapter is https://www.amazon.com/dp/B0CBJXWL1P.
- The driver installed is 2.18.1-1
- I get a maximum of 45 MB/s transfer rate.
- I also can't figure out where jumbo rates are, if that makes any difference.

I don't know if I am missing anything obvious :|
Is there a step that I missed or something that I am not seeing?

**Comment by Littlebo81** [comment]
Had same problem here,  turned out those C to A cheapo adapters are mostly USB 2.0 like [this](https://www.amazon.ca/dp/B08R6XGBTZ) one    All u need is to get a real deal  USB 3.0 piece like [this](https://www.amazon.ca/gp/product/B0CFVNGNCX) one  and you should be good. Read 1 star feedbacks, if it wont do over 2.0 ppl gna usually mention it there.  One more thing here, many chineseum 3.0  3.2  etc  adapters dont mirror all the contacts for both sides so they gna do 3.0 only on 1 side, if you cant get high speed then just flip C male in the adapter.

**Reply by bb-qq** [reply]
As pointed out in the previous comment, there may be a problem with the USB CtoA adapter; you can use the lsusb command to see if it is working with USB 3.0.
Please read here for more information.
https://github.com/bb-qq/r8152/wiki/Troubleshooting#check-the-protocol-version-of-the-usb-connection

**Reply by johnrues** [reply, marked as answer]
Thank you both and yes it turned out that one of the brand new ethernet cables that I had bought at the same time was this issue.
Everything works really well and is stable now.
@bb-qq thank you for the fantastic job that you're doing!

**Reply by bb-qq** [reply]
Since the problem seems to be resolved, I will close this thread.
